tools/bump_version.sh

```diff
@@ -5,9 +5,14 @@
 set -e
 
 # This script is used to update Asterinas version numbers in all relevant files in the repository.
-# Usage: ./tools/bump_version.sh <new_version>
+# Usage: ./tools/bump_version.sh [increment_type]
+# Increment_type can be one of: patch, minor, or major.
+# If not provided, the default is 'minor'.
```

Comment:
If not provided, the script should print the help message. This gives the user a chance to get to learn the correct usage of this script. This prevents a user from updating the version numbers in many files mistakenly.

```
# Usage: ./tools/bump_version.sh bump_type
#
# The bump_type argument must be either "patch", "minor", or "major",
# which instructs the script to increment the patch, minor, and major part
# of the version number, respectively.
```